pkg/volume/gce_pd/gce_util.go

```diff
@@ -148,61 +147,45 @@ func (gceutil *GCEDiskUtil) CreateVolume(c *gcePersistentDiskProvisioner) (strin
 		return "", 0, nil, "", fmt.Errorf("claim.Spec.Selector is not supported for dynamic provisioning on GCE")
 	}
 
+	var activezones sets.String
+	activezones, err = cloud.GetAllCurrentZones()
+	if err != nil {
+		glog.V(2).Infof("error getting zone information from GCE: %v", err)
```

Comment:
does this need to be logged since it will be returned as an error?

Reply:
This was in the original code but unnecessary. Will remove.